### ISSUE TITLE
New translation: Greek

### ISSUE THREAD

**Issue by cakrit** [squidfunk/mkdocs-material#986]
## Instructions

1. Check, if your language is already available: [here](http://bit.ly/2DCzaL0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "gr",
  "clipboard.copy": "Αντιγραφή",
  "clipboard.copied": "Αντιγράφηκε",
  "edit.link.title": "Επεξεργασία αυτής της σελίδας",
  "footer.previous": "Επόμενη",
  "footer.next": "Προηγούμενη",
  "meta.comments": "Σχόλια",
  "meta.source": "Πηγή",
  "search.language": "gr",
  "search.placeholder": "Αναζήτηση",
  "search.result.placeholder": "Πληκτρολογήστε για να αρχίσει η αναζήτηση",
  "search.result.none": "Δε βρέθηκαν αντίστοιχα αρχεία",
  "search.result.one": "1 αντίστοιχο αρχείο",
  "search.result.other": "# αντίστοιχα αρχεία",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Μετάβαση στο περιεχόμενο",
  "source.link.title": "Μετάβαση στο αποθετήριο (repository)",
  "toc.title": "Πίνακας περιεχομένων"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Added in 74c473e. Will be part of the next release.

**Comment by squidfunk** [comment]
Released as part of 4.0.0.
